proto/substrait/algebra.proto

```diff
@@ -131,14 +131,38 @@ message ReadRel {
       message ArrowReadOptions {}
       message OrcReadOptions {}
       message DwrfReadOptions {}
+      message TextReadOptions {
+        // The character(s) used to separate fields.  Common values are comma,
+        // tab, and pipe.
+        string field_delimiter = 1;
+        // The maximum number of bytes to read from a single row.  If a row
+        // exceeds this limit the resulting behavior is undefined.
+        uint64 max_block_size = 2;
```

Comment:
At first I thought this was spark specific (it is not in pandas) but I see that duckdb's csv reader does have an equivalent property so I think it's fine.

Minor nit: Why not `max_line_size`?

Reply:
Changed.